17thblog/logIn.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+include('config/config.php');
+include('config/init.php');
+
+echoHeader("Log In", "Log In");
+
+if(isset($_SESSION['Username']) && !empty($_SESSION['Username'])) {
+	echo "
+	<div class='textStyle form'>
+		**You are already logged in as $_SESSION[Username]!**
+		<br>
+		Click here to Log Out:
+		<a href='loggedOut.php'>Log Out</a>
+	</div>";
+}
+
+//form
+$Errors = array();
+
+if(isset($_REQUEST['logIn'])){ //
+
+	validateFormField('Username');
+	validateFormField('Password');
+
+	if(sizeof($Errors) == 0){
+
+		if(verifyUser($_REQUEST['Username'], $_REQUEST['Password']) == true) {
```

Comment:
The `== true` part isn't really necessary here. All `if` statements are evaluating if a statement is true, so this is like saying `if(true == true)` rather than just saying `if(true)`